app/tests/TestCase/Controller/GatheringsControllerTest.php

```
@@ -564,6 +564,69 @@ public function testCalendarGridDataReturnsMoreThanDefaultPageLimit(): void
+                'filters' => [[
+                    'field' => 'branch_id',
+                    'operator' => 'in',
+                    'value' => [(string)$branches[0]->public_id],
+                ]],
+            ]),
+        ], $member);
+        $this->assertInstanceOf(GridView::class, $gridView);
+        $this->authenticateAsMember(self::TEST_MEMBER_BRYCE_ID);
+
+        $this->get('/gatherings/calendar-grid-data?' . http_build_query([
+            'view_id' => $gridView->id,
```

PHPStan (2.2.8)

[error] Cannot access property $id on App\Model\Entity\GridView|false.


(property.nonObject)